wikichange/src/content/content.js

```diff
@@ -319,18 +345,18 @@ const highlightContentUsingNodes = (context, color) => {
  * Creates a text container with information about deletions side by side with the graph
  */
 const renderDeleteAlert = () => {
```

Comment:
I don't like how it looks stretched on the bottom of the graph - maybe removing the border will be better? Or making it narrower, idk. Or maybe adding some padding-top will help (too close to the date picker)

Reply:
I will make the box smaller and add some padding to seperate the box to the date chooser.  If we add more text, the box might look less stretched so I will make changes for further review.